lib/publiccloud/provider.pm

```diff
@@ -248,6 +249,42 @@ sub destroy_instance {
     $self->ipa(cleanup => 1, running_instance_id => $instance->instance_id);
 }
 
+sub _get_config {
+    my ($self) = @_;
+
+    unless ($self->{rest_config}) {
+        my $config_file = $ENV{OPENQA_CONFIG} || '/etc/openqa';
+        $config_file .= '/publiccloud.ini';
+        my $cfg = Config::IniFiles->new(-file => $config_file)
```

Comment:
again good enough for POC but later we need to get more sane error from `Config::IniFiles` like :
No permissions to access file 
File not exists
Wrong format and so on